charts/argus/Chart.yaml

```diff
@@ -6,9 +6,9 @@ maintainers:
   - email: argus@logicmonitor.com
     name: LogicMonitor
 name: argus
-version: 8.3.0-rc03
+version: 9.0.0-rc02
 home: https://logicmonitor.github.io/helm-charts-qa
-appVersion: v12.4.0-rc01
+appVersion: v13.0.0-rc02
```

Comment:
Any specific reason for directly bumping to rc02?

Reply:
@gurusiddha we had already published v13.0.0-rc01 couple of weeks back.
Reference PR https://github.com/logicmonitor/helm-charts-qa/pull/326/files#diff-b5c12b758b350a6daf75f9265e6ac07ee07904224a764f18c48fa4da74886c4dR11